daq_macros.py

```diff
@@ -3098,6 +3098,38 @@ def zebraCamDaq(zebra,angle_start,scanWidth,imgWidth,exposurePeriodPerImage,file
   
   setPvDesc("lowMagTrigMode",0)
 
+def zebraDaqBluesky(flyer, angle_start, scanWidth, imgWidth, exposurePeriodPerImage, filePrefix, data_directory_name, file_number_start, scanEncoder=3, changeState=True):
```

Comment:
Is this entirely replacing the function below? If so, it would be good to remove it to clean up. If not, do we need to change other things in there, such as changing daq_lib.setRobotGovState() to gov_lib.setGovRobot()?

Reply:
I think zebraDaq is being called from several places, so I only want to make this particular one use the new Bluesky one. I would like to make sure everything works, including where to put RE(), and sending references to the flyer object before I start cleaning up daq_macros.